dap_aria_mapping/pipeline/app_tables/emergence_table.py

```diff
@@ -0,0 +1,74 @@
+from dap_aria_mapping.getters.openalex import get_openalex_topics, get_openalex_works
+from dap_aria_mapping.getters.patents import get_patent_topics, get_patents
+from dap_aria_mapping.utils.app_data_utils import count_documents, expand_topic_col
+import polars as pl
+import pandas as pd
+import boto3
+import io
+#from nesta_ds_utils.loading_saving.S3 import upload_obj
+from dap_aria_mapping import BUCKET_NAME, logger
+
+if __name__ == "__main__":
+
+    logger.info("Loading patent data")
+    patents_with_topics = get_patent_topics(tax = "cooccur", level = 3)
```

Comment:
Do I understand correctly, for this particular chart we're using only `level=3` data?

Reply:
Ah I see, you're adding the higher levels as well and then aggregate when needed?

Reply:
Yeah - I used the bottom level that I was considering to iterate through, and then just aggregated at the end